spring-cloud-app-broker-core/src/main/java/org/springframework/cloud/appbroker/deployer/BackingService.java

```diff
@@ -120,6 +132,22 @@ public void setRebindOnUpdate(boolean rebindOnUpdate) {
 		this.rebindOnUpdate = rebindOnUpdate;
 	}
 
+	/**
+	 * Returns the requested updated plan if any. This enables to not trigger backing service when no plan update was
+	 * requested.
+	 */
+	public String getUpdatedPlanIfAny() {
+		if (getPlan() == null) {
```

Comment:
Can we update the tests instead of adding a branch in the code that will never be hit?

Reply:
Suspended until discussion in https://github.com/spring-cloud/spring-cloud-app-broker/pull/290#issuecomment-567106156 to remove the filtering CloudFoundryAppDeployer, and therefore this field would be removed